src/meadowrun/run_job_local.py

```diff
@@ -304,8 +310,62 @@ def _prepare_py_function(
     )
 
 
+_TASK_WORKER_PATH = str(
+    (
+        pathlib.Path(__file__).parent / "func_worker" / "__meadowrun_task_worker.py"
+    ).resolve()
+)
+
+
+async def _prepare_py_agent(
+    job: Job, io_folder: str, is_container: bool
+) -> _JobSpecTransformed:
+    """
+    Creates files in io_folder for the child process to use and returns
+    _JobSpecTransformed. We use __meadowrun_task_worker to start the function in the
+    child process.
+    """
+
+    if not is_container:
+        worker_path = _TASK_WORKER_PATH
+        io_path_container = os.path.join(io_folder, job.job_id)
+    else:
+        worker_path = (
+            f"{MEADOWRUN_CODE_MOUNT_LINUX}{os.path.basename(_TASK_WORKER_PATH)}"
+        )
+        io_path_container = f"{MEADOWRUN_IO_MOUNT_LINUX}/{job.job_id}"
+
+    server = await agent_start_serving("0.0.0.0" if is_container else "127.0.0.1")
```

Comment:
Would replacing `0.0.0.0` with `host.docker.internal` work? Slightly more secure I think?

Reply:
No that doesn't work. agent is outside the container, so won't have host.docker.internal. It's annoying but the 0.0.0.0 is the only thing I've been able to get working; there's some rumours that if you forward any port it also works, but it hasn't for me. Also strange is this works locally with localhost, but not on EC2.

Reply:
Got it, makes sense